### PR TITLE
feat: Handle query execution manually for biguery

### PR DESCRIPTION
**Pull request by chamini2**

### CODE REVIEW

File: src/faldbt/lib.py [hunk 1]
```diff
@@ -558,6 +561,32 @@ def _existing_or_new_connection(
 
 
 # Adapter: BigQuery
+def _bigquery_execute_sql(
+    adapter: BaseAdapter, sql: str, open_conn: bool
+) -> Tuple[AdapterResponse, pd.DataFrame]:
+    assert adapter.type() == "bigquery"
```
Comment (mederka) [inline src/faldbt/lib.py:567]: Should we have a message associated with this assertion?

File: src/faldbt/lib.py [hunk 2]
```diff
@@ -558,6 +561,32 @@ def _existing_or_new_connection(
...
+
+    import google.cloud.bigquery as bigquery
+
+    # HACK: we need to include uniqueness (UUID4) to avoid clashes
+    name = "bigquery:execute_sql:" + str(hash(sql)) + ":" + str(uuid4())
+    with _existing_or_new_connection(adapter, name, open_conn):
```
Comment (mederka) [inline src/faldbt/lib.py:573]: This doesn't cause conflicts if two execute_sql calls are running simultaneously? Or if a write is running?

**Review by mederka — APPROVED** [review]
This is very nice, @chamini2 !

**Comment by chamini2** [comment]
https://github.com/fal-ai/fal/runs/7135398253?check_suite_focus=true

**Comment by chamini2** [comment]
Worked https://github.com/fal-ai/fal/actions/runs/2593019997

**Pull request merged**